x-pack/plugins/enterprise_search/public/applications/shared/setup_guide/cloud/instructions.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { EuiPageContent, EuiSteps, EuiText, EuiLink, EuiCallOut } from '@elastic/eui';
+
+import { CLOUD_DOCS_PREFIX, ENT_SEARCH_DOCS_PREFIX } from '../../constants';
+
+interface Props {
+  productName: string;
+  cloudDeploymentLink?: string;
+}
+
+export const CloudSetupInstructions: React.FC<Props> = ({ productName, cloudDeploymentLink }) => (
```

Comment:
context for why this file is in a `setup_guide/cloud/` subfolder: it used to have extra associated sub-files such as screenshot images and an extra SCSS file for handling those images, but after a discussion w/ Casey & Davey we decided to scrap the images as being really make accessible & keep up to date with, e.g. dark/light themes, Amsterdam changes, etc.

I still kept this in a folder mostly out of laziness but I can pull it back out to the top level if people strongly prefer, just lmk